From 65a29965a8eb55eb855090bb7548997aeca2b9a1 Mon Sep 17 00:00:00 2001
From: Jaroslav Kysela <jkysela@redhat.com>
Date: Wed, 3 Sep 2014 07:16:33 -0400
Subject: [alsa] hda/conexant: Add analog loopback mixing to CX20549

Message-id: <140972823573-74-patchset-jkysela@redhat.com>
Patchwork-id: 90774
O-Subject: [RHEL 7.1 PATCH 073/442] ALSA: hda/conexant - Add analog loopback mixing to CX20549
Bugzilla: 1112200
RH-Acked-by: John Linville <linville@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Ivan Vecera <ivecera@redhat.com>

CX20549 has an aamixer widget at NID 0x17.  Let's enable it.

Signed-off-by: Takashi Iwai <tiwai@suse.de>

(picked from kernel: 70540e245c727114d0477f00ec7d83a2d2fe666d)
rhbz#1112200
Signed-off-by: Jaroslav Kysela <jkysela@redhat.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/sound/pci/hda/patch_conexant.c b/sound/pci/hda/patch_conexant.c
index e4b98d6..6b2c11d 100644
--- a/sound/pci/hda/patch_conexant.c
+++ b/sound/pci/hda/patch_conexant.c
@@ -3423,6 +3423,7 @@ static int patch_conexant_auto(struct hda_codec *codec)
  switch (codec->vendor_id) {
  case 0x14f15045:
   codec->single_adc_amp = 1;
+  spec->gen.mixer_nid = 0x17;
   snd_hda_pick_fixup(codec, cxt5045_fixup_models,
        cxt5045_fixups, cxt_fixups);
   break;
-- 
1.7.1